pybv/io.py

```diff
@@ -125,6 +128,9 @@ def write_brainvision(*, data, sfreq, ch_names, fname_base, folder_out,
 
     """
     # Input checks
+    if not isinstance(overwrite, bool) and overwrite not in [0, 1]:
```

Comment:
I would only allow `True` and `False` here. What's the point of allowing also 0 and 1? IMO it is irrelevant whether or not Python represents `True` and `False` as `int` internally.

Reply:
I had this because many people use 0 and 1 as a shorthand for False and True, and isinstance wouldn't accept that, see:

```
In [1]: isinstance(1, bool)
Out[1]: False
```

Reply:
Yeah, I don't like it. They should be using `True` and `False`.

Reply:
okay --> eefd93ef5a37808439883af561f6dd9bcb46ac5b